backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -16,9 +15,13 @@
 import java.util.Set;
 
 @Entity
-@Table(name = "cars")
-@Getter
 @NoArgsConstructor
+@AllArgsConstructor
```

Comment:
All argument constructor is needed?

Reply:
I checked and it's actually needed because @ Builder requires it. You can read more here: [https://stackoverflow.com/questions/34241718/lombok-builder-and-jpa-default-constructor](url)